pdi-dataservice-client/src/main/java/org/pentaho/di/trans/dataservice/jdbc/ThinConnection.java

```diff
@@ -556,7 +556,9 @@ public ThinConnection build() throws SQLException {
     }
 
     private URI baseUri() throws SQLException {
-      boolean isSecure = "true".equalsIgnoreCase( arguments.get( ARG_ISSECURE ) );
+      boolean isSecure = "true".equalsIgnoreCase( arguments.get( ARG_ISSECURE ) )
+              || uri.getPort() == 8443
```

Comment:
probably a more reliable option would be: `"https".equalsIgnoreCase(` uri.getScheme() )`.
WDYT, @pamval ?